test/integration/targets/binary_modules/host_vars/testhost

```diff
@@ -0,0 +1,3 @@
+system: "{{ ansible_system|lower }}"
```

Comment:
My previous advice on this was off a bit. Use `group_vars/all` so that it will apply to the windows hosts as well.